src/GNDStk/HDF5/test/HDF5.test.cpp

```diff
@@ -0,0 +1,17 @@
+
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+using namespace njoy::GNDStk;
+
+// -----------------------------------------------------------------------------
+// SCENARIO
+// -----------------------------------------------------------------------------
+
+SCENARIO("Testing GNDStk HDF5") {
+
+   /// fixme adapt code from the corresponding JSON test
+
+}
```

Comment:
I assume you are planning on adding tests?

Reply:
Yep; see other comment.